### PR TITLE
Add Finance Director role

### PR DESCRIPTION
**Pull request by gabrielamatias**
@jacobpgn : please review.

### CODE REVIEW

File: app/pages/about/jobs/positions/finance-director.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/finance-director.js]: Snip blank line

File: app/router/routes.js [hunk 2]
```diff
@@ -55,6 +55,8 @@ import LegalAssistant from '../pages/about/jobs/positions/legal-assistant.js';
 import ComplianceManager from '../pages/about/jobs/positions/compliance-manager.js';
 import PartnershipsManagerSME from '../pages/about/jobs/positions/partnerships-manager.js';
 import PartnershipsManagerEnterprise from '../pages/about/jobs/positions/partnerships-manager-enterprise.js';
+import FinanceDirector from '../pages/about/jobs/positions/finance-director.js';
+
```
Comment (jacobpgn) [inline app/router/routes.js]: Snip blank line

**Comment by gabrielamatias** [comment]
@jacobpgn : good to go? I snipped the blank line and for some reason it's not showing there.

**Comment by jacobpgn** [comment]
👌 👍

**Pull request merged**